apps/gpc-prover-client/.env.example

```diff
@@ -1,3 +1,6 @@
 # URLs to other zukyc applications
 NEXT_PUBLIC_POD_ISSUER_CLIENT_URL=http://localhost:3000
 NEXT_PUBLIC_GPC_VERIFIER_URL=http://localhost:3001
+
+# URL to gpc artifacts
```

Comment:
This needs documentation about how it would work in a real deployment?
If the prover and verifier run on different servers, they may have CORS issues downloading.

Reply:
I have CORS setup in the artifacts server. The prover and verifier are currently running on different servers, it works directly without needing to change things.